### ISSUE TITLE
Add suggested severity field to the a11y issue template

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#16489]
This is missing and makes it more likely that we miss adding the proper metadata (effort, labels, etc.) on these issues.

It might also be worth exploring if there is support in the issue templates for automatically setting Github Projects metadata fields like we can already do for labels.

**Comment by Gururajj77** [comment]
closed via #16641
